src/content/self-hosted/install/releases.mdx

```diff
@@ -7,14 +7,22 @@ id: releases
 
 ## 1.15.0
 
-TBD
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
 
 ### Improvements
```

Comment:
We have to include the version of the CLI included as part of the release. We usually include it as feature I think